docs/src/pages/roadmap.tsx

```diff
@@ -231,18 +234,31 @@ const roadmap: Item[] = [
 ];
 
 const milestones: Item[] = [
+  withRelease({
+    icon: mdiFaceRecognition,
+    title: 'Metadata Face Import',
+    description: 'Read face metadata in Digikam format during metadata extraction',
+    release: 'v1.114.0',
+  }),
   withRelease({
     icon: mdiTagMultiple,
     iconColor: 'orange',
     title: 'Tags',
     description: 'Tag your photos and videos',
     release: 'v1.113.0',
   }),
+  withRelease({
+    icon: mdiFolderSync,
+    iconColor: 'green',
+    title: 'Ablume sync (mobile)',
+    description: 'Sync or mirror an album from your phone to the Immich server',
+    release: 'v1.113.0',
+  }),
   withRelease({
     icon: mdiFolderMultiple,
     iconColor: 'brown',
     title: 'Folders',
-    description: 'View your photos and videos in folders',
+    description: 'View your external photos and videos in folders',
```

Comment:
That does not only apply to external libraries.

Reply:
Maybe it's worth mentioning in some way that these are external folders as well?
When I first read it it looked like it was an addition of something "simple".

I know a lot of people have been waiting for this feature, maybe for those following the roadmap it can be confusing.

Reply:
> Browse your photos and videos in their folder structure

?

Reply:
Not a huge fan tbh, idk....

Reply:
> > Browse your photos and videos in their folder structure
> 
> ?

In fact I think it explains this feature well.